cypress/mocks/migrated-from-pact.json

```diff
@@ -154,6 +154,18 @@
         "headers": { "Content-Type": "application/json" }
       }
     },
+    {
+      "name": "A request to get tasks for a case",
+      "request": {
+        "method": "GET",
+        "url": "/lpa-api/v1/cases/800/tasks?filter=status%3ANot+started%2Cactive%3Atrue&limit=99"
+      },
+      "response": {
+        "status": 200,
+        "headers": { "Content-Type": "application/json" },
+        "body": "{\"tasks\":[{\"id\":1,\"name\":\"Review reduced fee eligibility\",\"assignee\":{\"displayName\":\"Super Team\"}},{\"id\":2,\"name\":\"Review application correspondence\",\"assignee\":{\"displayName\":\"Marvellous Team\"}},{\"id\":3,\"name\":\"Another task\",\"assignee\":{\"displayName\":\"Super Team\"}}]}"
+      }
+    },
```

Comment:
Since it's only used once I'd suggest creating it with `cy.addMock()` within the test

Reply:
I didn't realise that was an option, thanks. Though I have to say I find it useful to be able to use the mock server for manual testing. Would adding it in the cypress test then make it available for manual tests?

Reply:
No the `cy.addMock` responses only work for the duration of the test they're running in. Interesting about using the mock server for testing, I prefer running it against actual Sirius so I can try things against a variety of resources.

Reply:
I would prefer to leave this, as it works both for cypress and manual testing.